DQM/CTPPS/plugins/CTPPSPixelDQMSource.cc

```diff
@@ -337,22 +341,16 @@ void CTPPSPixelDQMSource::beginLuminosityBlock(edm::LuminosityBlock const& lumiS
 void CTPPSPixelDQMSource::analyze(edm::Event const& event, edm::EventSetup const& eventSetup)
 {
   ++nEvents;
-  int RPactivity[2][NRPotsMAX];
-  for(int arm = 0; arm <2; arm++) { 
-    for(int rp=0; rp<NRPotsMAX; rp++) {
-      RPactivity[arm][rp] = 0;
-    }
-  }
-  for(int ind=0; ind<2*3*NRPotsMAX; ind++) { 
-    for(int p=0; p<NplaneMAX; p++) {
-      HitsMultPlane[ind][p] = 0;
-    }
-  }
-  for(int ind=0; ind<2*3*NRPotsMAX*NplaneMAX; ind++)  {
-    for(int rp=0; rp<NROCsMAX; rp++) {
-      HitsMultROC[ind][rp] = 0;
-    }
-  }
+ int RPactivity[NArms][NRPotsMAX], digiSize[NArms][NRPotsMAX];
+ for(int arm = 0; arm <2; arm++) for(int rp=0; rp<NRPotsMAX; rp++)
+   RPactivity[arm][rp] = digiSize[arm][rp] = 0;
+ 
+ for(int ind=0; ind<2*3*NRPotsMAX; ind++) 
```

Comment:
that is initialization during compiling stage, but I need
 initialization in every analyzed event...
 or this assumption is wrong ?
 thanks,
   Vladimir
for today this PR is already obsolete (overridden by PR 19670/19895)